admin/workspace-providers/deployment.md

```diff
@@ -119,6 +122,10 @@ when communicating with the Coder deployment.
    Values]("https://github.com/cdr/enterprise-helm/blob/workspace-providers-envproxy-only/README.md")
    for more details.
 
+   > **NOTE**: If the hostname provided is using https you must ensure the
```

Comment:
```suggestion
For installations using HTTPS, you must [ensure the deployment has a valid certificate](../../guides/ssl-certificates/index.md).
```

This is shorter and allows us to drop the SSL acronym. I still think it's a shame that we're still (as an industry) using the long-outdated acronym SSL rather than its accessor, TLS, but I digress :man_shrugging: